cmd/bootstrap/cmd/check_machine_account.go

```diff
@@ -0,0 +1,100 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"time"
+
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc"
+
+	sdk "github.com/onflow/flow-go-sdk"
+	"github.com/onflow/flow-go-sdk/client"
+	model "github.com/onflow/flow-go/model/bootstrap"
+	"github.com/onflow/flow-go/module/epochs"
+)
+
+var (
+	flagAccessAPIAddress string
+)
+
+var checkMachineAccountCmd = &cobra.Command{
+	Use:   "check-machine-account",
+	Short: "checks a machine account configuration",
+	Run:   checkMachineAccountRun,
+}
+
+func init() {
+	rootCmd.AddCommand(checkMachineAccountCmd)
+
+	checkMachineAccountCmd.Flags().StringVar(&flagAccessAPIAddress, "access-address", "", "network address of an Access Node")
+	_ = machineAccountCmd.MarkFlagRequired("address")
```

Comment:
Good catch, we should be handling this, otherwise this can silently fail to mark a flag required. Added a helper for this in https://github.com/onflow/flow-go/pull/1233/commits/aa4135fe51665822bf6f4f30f7ecbc0be7a4c971